p2p-specs/p2p-interface.md

```diff
@@ -606,12 +617,10 @@ class UserOp(Container):
     signature: bytes
 ```
 
-#### `UserOperationsWithEntryPoint`
+#### `VerifiedUserOperation`
 
 ```python
-class UserOperationsWithEntryPoint(Container):
-    entry_point_contract: Address
+class VerifiedUserOperation(Container):
+    user_operation: UserOp
```

Comment:
see my [comment here](https://hackmd.io/@dancoombs/r1u4CcB-p?comment=58abf893-f5fd-4e9c-a449-68c5cee0bb90&utm_source=comment-card&utm_medium=icon), why I think we can't remove entrypoint address

Reply:
Added back.